build_tools/cmake/iree_run_module_test.cmake

```diff
@@ -0,0 +1,276 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Utility function to check expected failure for different platforms.
+function(platform_xfail OUTPUT_XFAIL XFAIL_ARGS)
+  set(${OUTPUT_XFAIL} FALSE PARENT_SCOPE)
+  if(XFAIL_ARGS)
+    if((CMAKE_SYSTEM_PROCESSOR STREQUAL "riscv64" AND
+        CMAKE_SYSTEM_NAME STREQUAL "Linux" AND
+        XFAIL_ARGS MATCHES "rv64") OR
```

Comment:
Linux is assumed unless it's -baremetal?

Reply:
Yes. We only distinguish `rv32-linux` and `rv32-baremetal` because RISC-V 32-bit has a baremetal build configuration.